lib/modules/manager/docker-compose/__fixtures__/docker-compose.3-tags.yml

```diff
@@ -0,0 +1,25 @@
+services:
```

Comment:
reduce to minimal (one service) and inline to test

Reply:
if i reduce the test case to one service, the docker compose file ist not a valid compose file anymore.
Should this ok?

You need minimum two services to write an valid extend and override configuration.